tools/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
We should rename this to something that indicates that this tool only works with .NET code.  Personally, I'd prefer that it also has a mention of "management" somewhere in there.   I'd prefer not to see data plane libraries using this auto-generated approach.